docs/docs/2.quick_start/2.1.installation/2.1.1.prerequisites.md

```diff
@@ -5,8 +5,44 @@ sidebar_label: "Prerequisites"
 
 # Prerequisites
 
-HwameiStor must be deployed on a Kubernetes platform. The hosts must meet the below requirements:
+## Kubernetes
 
-- Kubernetes version: `1.18+`
-- LVM is installed
-- Have at least one unused drive
+1. Kubernetes version: `1.18+`
+1. CoreDNS is deployed
+
+## Host
+
+### Linux Distribution
+
+1. CentOS `7.5+`
+1. Kylin `V10`
+
+### Processor Architecture
+
+1. x86_64
+1. ARM64
+
+### Package Dependencies
+
+1. The latest `LVM2`
+1. For HA features, `kernel-devel` must be installed and match the version of the operating `kernel`
+
+```bash
+# CentOS and Kylin
+$ yum install -y lvm2
+$ yum install -y kernel-devel-$(uname -r)
+```
+
+### Data Disk
+
+HwameiStor supports `HDD`, `SSD` and `NVMe`.
+
+For test, each host must have at least one unused drive with a minimal size of 10GiB.
+
+For production, it is recommended to have a least one unused drive, protected by RAID1 or RAID5/6, with a minimal size of 200GiB.
+
+### Network
+
+For test, a simple 1Giga TCP/IP network is adequate.
+
+For production, and especially with HA features, it is recommended to have a redundant 10Giga TCP/IP network.
```

Comment:
Recommended to have a redundant 10Giga TCP/IP network, if enable the HA features .